### PR TITLE
[Feat] Added Flagr version to UI

### LINKED ISSUE #287: [feat] Add Flagr version to UI
Add a display of the flagr version number somewhere in the flagr UI.  This would help issue reporters be able to find that quickly to include in reports. 

## Expected Behavior
Flagr version number displays in the web UI.  

## Current Behavior
Flagr version cannot be discovered in the web UI

## Possible Solution
It might be part of the top-level bar alongside 'API', 'Docs', etc, or there could be an 'About' item there with a corresponding page that includes the version and maybe some other info.

## Steps to Reproduce (for bugs)
Run flagr, go to website.  Observe version is not visible.

## Context
Had to dig a little bit to find the version number for including in the github issue template

## Your Environment

* Version used (`flagr version`): 1.1.4

### PR DESCRIPTION
**Pull request by wesleimp**
Added Flagr version to UI

## Description
Added Flagr version to UI on the top-level bar alongside `Flagr`

## Motivation and Context
Closes #287 

## How Has This Been Tested?
Run flagr, go to website and observe version on the top.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -21,18 +21,21 @@ github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRF
 github.com/apache/thrift v0.12.0/go.mod h1:cp2SuWMxlEZw2r+iP2GNCdIi4C1qmUzdZFSVb+bacwQ=
```
Comment (zhouzhuojie) [inline go.sum]: let's move the go.sum update to another PR, I don't think UI changes need to check in go.sum

File: browser/flagr-ui/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,8670 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (zhouzhuojie) [inline browser/flagr-ui/yarn.lock]: we removed yarn, so no need to add this file

File: go.sum (hunk 1 above)
Comment (wesleimp) [reply go.sum]: To be honest, I'm not sure why `go.sum` has changed.

I'll remove this file from this pr

**Comment by zhouzhuojie** [comment]
btw, the h3 of other links are no longer white anymore. Wondering if you can tweak the color a little bit.

```
diff --git a/browser/flagr-ui/src/App.vue b/browser/flagr-ui/src/App.vue
index eec25af..9b524d5 100644
--- a/browser/flagr-ui/src/App.vue
+++ b/browser/flagr-ui/src/App.vue
@@ -5,7 +5,7 @@
         <el-col :span="6">
           <router-link :to="{name: 'home'}">
             <div class="logo-container">
-              <h2 class="logo">Flagr</h2>
+              <h3 class="logo">Flagr</h3>
               <div>
                 <span class="version">v{{ version }}</span>
               </div>
@@ -29,7 +29,7 @@
 <script>
 const version = require("../package.json").version || "1.0.0"
 export default {
-  name: 'app', 
+  name: 'app',
   data: () => ({
     version
   }),
@@ -77,24 +77,22 @@ ul {
 
   .navbar {
     background-color: #74E5E0;
+    h3 {
+      color: white;
+      &:hover {
+        color: #f7f7f7;
+      }
+    }
+
     .logo-container {
       display: flex;
       justify-content: center;
       align-items: center;
       font-weight: bold;
-
-      h2 {
-        color: white;
-        margin-left: 2em;
-        margin-right: 10px;
-        &:hover {
-          color: #f7f7f7;
-        }
-      }
-
+      color: white;
       span {
+        margin-left: 10px;
         font-size: 12px;
-        color: #551a8b;
       }
     }
     a {

```

image

**Comment by wesleimp** [comment]
> btw, the h3 of other links are no longer white anymore. Wondering if you can tweak the color a little bit.
> 
> ```
> diff --git a/browser/flagr-ui/src/App.vue b/browser/flagr-ui/src/App.vue
> index eec25af..9b524d5 100644
> --- a/browser/flagr-ui/src/App.vue
> +++ b/browser/flagr-ui/src/App.vue
> @@ -5,7 +5,7 @@
>          <el-col :span="6">
>            <router-link :to="{name: 'home'}">
>              <div class="logo-container">
> -              <h2 class="logo">Flagr</h2>
> +              <h3 class="logo">Flagr</h3>
>                <div>
>                  <span class="version">v{{ version }}</span>
>                </div>
> @@ -29,7 +29,7 @@
>  <script>
>  const version = require("../package.json").version || "1.0.0"
>  export default {
> -  name: 'app', 
> +  name: 'app',
>    data: () => ({
>      version
>    }),
> @@ -77,24 +77,22 @@ ul {
>  
>    .navbar {
>      background-color: #74E5E0;
> +    h3 {
> +      color: white;
> +      &:hover {
> +        color: #f7f7f7;
> +      }
> +    }
> +
>      .logo-container {
>        display: flex;
>        justify-content: center;
>        align-items: center;
>        font-weight: bold;
> -
> -      h2 {
> -        color: white;
> -        margin-left: 2em;
> -        margin-right: 10px;
> -        &:hover {
> -          color: #f7f7f7;
> -        }
> -      }
> -
> +      color: white;
>        span {
> +        margin-left: 10px;
>          font-size: 12px;
> -        color: #551a8b;
>        }
>      }
>      a {
> ```
> 
> image

Sorry, my bad!

I think now the changes are ok

**Pull request merged**